src/lib/snark_work_lib/metrics.ml

```diff
@@ -96,13 +42,19 @@ let emit_single_metrics ~(single_spec : _ Single_spec.Poly.t)
           Mina_metrics.(
             Counter.inc Cryptography.snark_work_nonzkapp_base_time_sec
               (Time.Span.to_sec elapsed)) ) ;
-      Base_snark_generated { transaction_type; elapsed }
+      [%log info]
+        "Base SNARK generated in $elapsed for $transaction_type transaction"
+        ~metadata:
+          [ ("elapsed", `Float (Time.Span.to_sec elapsed))
+          ; ("transaction_type", Transaction_type.to_yojson transaction_type)
```

Comment:
For zkapp command it'd be useful to include codification of account updates, like `SPPS` (signed-proof-proof-signed).

We could define this function somewhere in `mina_base` so it can later be used in other parts.

Just to confirm: am I right that log here will happen after we compute all of the underlying proofs for the zkapp?